attachments/static/attachments/js/attachments.js

```diff
@@ -12,7 +12,7 @@
         
         var refresh = function() {
         	var data = {};
-        	if (settings.container.hasClass('bind-form-on-refresh')) {
+        	if (!! settings.container && settings.container.hasClass('bind-form-on-refresh')) {
```

Comment:
Weird, first time I've actually seen !! used. Doesn't the && make it evaluate in boolean context anyway? Not a big deal, just curious if there's a difference.